mesheryctl/internal/cli/root/system/config.go

```diff
@@ -329,12 +331,86 @@ mesheryctl system config minikube
 	},
 }
 
+var okeConfigCmd = &cobra.Command{
+	Use:   "oke",
+	Short: "Prepare a kubeconfig for Meshery to connect to an OKE cluster",
+	Long:  `Using the Oracle Cloud Infrastructure CLI, prepare a kubeconfig for Meshery to connect to an Oracle Kubernetes Engine cluster`,
+	Example: `
+// Prepare a kubeconfig for Meshery to create a connection to an Oracle Kubernetes Engine (OKE) cluster.
+mesheryctl system config oke
+
+// Optionally, use a specific Meshery API token other than the token specified in your current context.
+mesheryctl system config oke --token auth.json
+	`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 1 {
+			return errors.New("more than one config name provided")
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		okeCheck := exec.Command("oci", "--version")
```

Comment:
This version number displays in stdout without context as to why it's being displayed. Please either remove it or add a note like:
```("Found `oci` CLI version %s", version)```